packages/amplify-codegen-e2e-tests/src/__tests__/graphql-documents-generator.test.ts

```diff
@@ -31,9 +32,10 @@ describe('GraphQL documents generator e2e tests', () => {
   });
 
   const schemaFileName = 'schema.graphql';
-  it('generates valid GraphQL documents for given schema', async () => {
+  // skip cypress test on windows
+  (isWindows() ? it.skip : it)('generates valid GraphQL documents for given schema', async () => {
```

Comment:
Do we want to run these on windows as well eventually? If yes, we should have a task for it.

Reply:
I've created tasks for the 2 disabled tests and the region issue.